Lesson Plans/Private/PPL.19 Navigation Systems and Other Equipment.md

```diff
@@ -30,7 +30,6 @@ Develop knowledge and ability to use onboard navigation equipment to enter and f
 6. ATC services and assistance available
 	1. Flight Following
 	2. ASR (airport surveillance radar) and PAR (precision approach radar) capabilities
-7. Inoperative Equipment
```

Comment:
Though this is a relevant lesson, it seems a bit too late in training to cover it. Decided instead to move the conversation to pre-solo and review pre-checkride